dev/com.ibm.ws.logging.osgi/src/com/ibm/ws/logging/internal/osgi/stackjoiner/ThrowableInfo.java

```diff
@@ -24,62 +24,60 @@
 public class ThrowableInfo {
 	
 	private static final TraceComponent tc = Tr.register(ThrowableInfo.class);
-
-    final String BASE_TRACE_SERVICE_CLASS_NAME = "com.ibm.ws.logging.internal.impl.BaseTraceService";
-    final String BASE_TRACE_SERVICE_METHOD_NAME = "printStackTraceOverride";
-
-    private Method btsMethod;
+	final String BASE_TRACE_SERVICE_CLASS_NAME = "com.ibm.ws.logging.internal.impl.BaseTraceService";
+	final String BASE_TRACE_SERVICE_METHOD_NAME = "printStackTraceOverride";
+	private Method btsMethod;
 
     
-    public ThrowableInfo(Instrumentation inst) {
-    	if (isEnabled()) {
+	public ThrowableInfo(Instrumentation inst) {
+		if (isEnabled()) {
 	    	Class<?> btsClass = retrieveClass(inst, BASE_TRACE_SERVICE_CLASS_NAME);
 	        if (btsClass != null) {
 				Method method = ReflectionHelper.getDeclaredMethod(btsClass, BASE_TRACE_SERVICE_METHOD_NAME, Throwable.class, PrintStream.class);
 				setBtsMethod(method);
 	        }
-    	}
-    }
-    public boolean isInitialized() {
-    	if (isEnabled()) {
-	    	if (getBtsMethod() == null) {
-	    		if (tc.isDebugEnabled())
-	    			Tr.debug(tc, "Stack joiner could not be initialized. Failed to reflect method " + BASE_TRACE_SERVICE_METHOD_NAME + " in " + BASE_TRACE_SERVICE_CLASS_NAME + ".");
-	        	return false;
-	        }
-	    	return true;
-    	}
-    	return false;
-    }
+		}
+	}
+	public boolean isInitialized() {
+		if (isEnabled()) {
+			if (getBtsMethod() == null) {
+				if (tc.isDebugEnabled())
+					Tr.debug(tc, "Stack joiner could not be initialized. Failed to reflect method " + BASE_TRACE_SERVICE_METHOD_NAME + " in " + BASE_TRACE_SERVICE_CLASS_NAME + ".");
+				return false;
+				}
+			return true;
```

Comment:
fix indent

Reply:
Fixed in [c526dc4](https://github.com/OpenLiberty/open-liberty/pull/16276/commits/c526dc432032c3e2466cddbf875e78e70c7de8da)